Documentation/proposals/202501-configuration-object-status-subresource.md

```diff
@@ -0,0 +1,192 @@
+## Status Subresource For Config-Based Resources
+
+* **Owners:**
+  * [yp969803](https://github.com/yp969803)
+* **Status:**
+  * `In-Progress`
+* **Related Tickets:**
+  * [#3385](https://github.com/prometheus-operator/prometheus-operator/issues/3335)
+* **Other docs:**
+  * NA
+
+> This proposal describes how we will extend the Prometheus operator configuration Custom
+> Resource Definitions (CRDs) with a Status subresource field.
+
+## Why
+
+This will enhance observability by allowing users to verify whether their configurations have been successfully applied to Workload.
```

Comment:
(nit)
```suggestion
This will allow users to verify whether their configurations have been successfully applied to the corresponding workload resources.
```